### ISSUE TITLE
 Could not load content: ERR_HTTP_RESPONSE_CODE_FAILURE

### ISSUE THREAD

**Issue by chemelli74** [custom-cards/bar-card#85]
Error logged while debugging with Chromium:

`DevTools failed to load SourceMap: Could not load content for https://xxxxxxxxx/hacsfiles/bar-card/index.m.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

- Home Assistant release with the issue: 0.110.3
- Browser and browser version: Chromium - Version 83.0.478.37 (Official build) (64-bit)
- Operating system: Windows 10

Simone

**Comment by chemelli74** [comment]
Latest build of HACS + latest of HA shows error in log:

`2020-06-16 17:41:41 ERROR (MainThread) [hacs] Tried to serve up '/config/www/community/bar-card/index.m.js.map' but it does not exist
`

File is indeed missing.

Simone

**Comment by Sjeff** [comment]
Same error with: 
  - Home Assistant Version 0.112.3
  - Chrome Version 83.0.4103.116
  - Home Assistant Supervised (Ubuntu Docker & Raspbian Docker)

Chrome debugging message:
```
DevTools failed to load SourceMap: Could not load content for https://xxx.xxx.xxx.xxx/hacsfiles/bar-card/index.m.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```
 
HA log:
```
Log Details (ERROR)
Logger: hacs
Source: custom_components/hacs/http.py:82
First occurred: 10:06:54 AM (2 occurrences)
Last logged: 10:06:54 AM

Tried to serve up '/config/www/community/bar-card/index.m.js.map' but it does not exist
```

**Comment by Gluwc** [comment]
Should be fixed in [3.1.7](https://github.com/custom-cards/bar-card/releases/tag/3.1.7).
